src/php/ext/grpc/php_grpc.c

```diff
@@ -57,21 +59,20 @@ ZEND_DECLARE_MODULE_GLOBALS(grpc);
 
 /* {{{ grpc_module_entry
  */
-zend_module_entry grpc_module_entry = {
-  STANDARD_MODULE_HEADER,
-  "grpc",
-  grpc_functions,
-  PHP_MINIT(grpc),
-  PHP_MSHUTDOWN(grpc),
-  PHP_RINIT(grpc),
-  NULL,
-  PHP_MINFO(grpc),
-  PHP_GRPC_VERSION,
-  PHP_MODULE_GLOBALS(grpc),
-  PHP_GINIT(grpc),
-  NULL,
-  NULL,
-  STANDARD_MODULE_PROPERTIES_EX};
+zend_module_entry grpc_module_entry = {STANDARD_MODULE_HEADER,
```

Comment:
Lots of formatting changes in this file too.

Reply:
reverted these files .... changes weren't intentional